### DISCUSSION TITLE
Current clickhouse-server version (21.3.2.5) doesn't work on Apple Silicon (M1)

### DISCUSSION THREAD

**Discussion by ccmaris** [plausible/analytics#1671] [Self-Hosted Support]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Describe the bug

# TLDR
`make clickhouse` fails on Apple Silicon M1

# Details
The current version of Clickhouse Server (21.3.2.5) doesn't work on Apple Silicon.

After doing some research, I realized that a newer version of Clickhouse Server (21.11) fixes this issue.

Can we please upgrade the Clickhouse Server?

### Expected behavior

`make clickhouse` should work on Apple Silicon

### Environment

```markdown
- OS: MacOS Monterey v12.1
- Platform: Apple Silicon M1
- Clickhouse Server: v21.3.2.5
```

**Comment by rajdesai24** [comment]
I am having the same issues with self hosting because of the clickhouse version in docker-compose up

**Reply by ccmaris** [reply]
I had to bump up the version locally and just make sure I don't commit that change.
